forge-gui/res/cardsfolder/i/invasion_plans.txt

```diff
@@ -2,7 +2,8 @@ Name:Invasion Plans
 ManaCost:2 R
 Types:Enchantment
 S:Mode$ MustBlock | ValidCreature$ Creature | Description$ All creatures block each combat if able.
-S:Mode$ Continuous | GlobalRule$ The attacking player chooses how each creature blocks each combat. | Description$ The attacking player chooses how each creature blocks each combat.
+R:Event$ DeclareBlocker | Layer$ Control | ReplaceWith$ DBUpdateChooser | Description$ The attacking player chooses how each creature blocks each combat.
```

Comment:
this shouldn't be RE that let's you choose
this matters especially if multiple _Master Warcraft_ would resolve

Reply:
Then Master Warcraft would need to be reworked too

The thing probably needs to use Card Timestamp logic?

Reply:
yea #3582